yarn.lock

```diff
@@ -9860,7 +9860,7 @@ strict-uri-encode@^1.0.0:
   resolved "https://registry.yarnpkg.com/strict-uri-encode/-/strict-uri-encode-1.1.0.tgz#279b225df1d582b1f54e65addd4352e18faa0713"
   integrity sha512-R3f198pcvnB+5IpnBlRkphuE9n46WyVl8I39W/ZUTZLz4nqSP/oLYUrcnJrw462Ds8he4YKMov2efsTIw1BDGQ==
 
-"string-width-cjs@npm:string-width@^4.2.0", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
+"string-width-cjs@npm:string-width@^4.2.0":
```

Comment:
Now you introduced changes to some package resolutions that haven't been there before the latest force-push :/

Reply:
Not sure why now `yarn` is changing the resolutions and adding all of that noise.

**I have edited now the `yarn.lock` manually.**

The `yarn.lock` is minimal and I also run `yarn install` to verify that the everything is ok.